docbook/Admin_Guide/en-US/config/auth.xml

```diff
@@ -109,13 +114,36 @@
 
 					<para>Examples of valid URI:</para>
 					<programlisting>
-ldap.example.com
 ldap://ldap.example.com/
 ldaps://ldap.example.com:3269/
 </programlisting>
 				</listitem>
 			</varlistentry>
 
+			<varlistentry>
+				<term>$g_ldap_use_starttls</term>
+
+				<listitem>
+					<para>A boolean indicating if the connection will attempt opportunistic
+					upgrade to a TLS connection (STARTTLS).  Be sure to use the ldap:// scheme
+					and not ldaps:// if setting this to ON.  For security, a failure aborts
+					the entire connection. Defaults to <emphasis>true</emphasis>.</para>
```

Comment:
After latest changes, change to `Defaults to ON`

Reply:
Done